the-new-developers/gatehopper
# Sohier/create cheatsheet

## Changes:

![output](https://user-images.githubusercontent.com/42362224/127756107-4f4fc9d0-4cbb-41d9-bd9b-8112201bd827.gif)

Added a cheat sheet, 70% cheat sheet and main menu. To accomplish this, added the capability to add buttons that use images on screen.

Closes #15
Closes #16
Closes #19
Closes #33

## How to test this PR:

Run on a web server and click the buttons on screen. The "CHEAT SHEET" button should take the user to an implemented scene of which would allow the user to return to main menu. Cheat sheet assets should be in /assets/textures

## Checklist:
